src/components/SocialLinks/SocialLinks.css

```diff
@@ -1,5 +1,10 @@
 .social-link {
   margin: 0px 10px;
+  transition: opacity 255ms ease;
+}
+
+.social-link:hover {
+  opacity: .8;
```

Comment:
@emmawedekind is it readable enough?

Reply:
An opacity of .8... hmm I don't think it is... let's keep opacity of 1 for a11y purposes

Reply:
@emmawedekind done, I've removed the opacity on hover